plugin/src/main/java/com/playmonumenta/scriptedquests/zones/zone/ZoneFragment.java

```diff
@@ -0,0 +1,254 @@
+package com.playmonumenta.scriptedquests.zones.zone;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.bukkit.Axis;
+import org.bukkit.util.Vector;
+
+import com.playmonumenta.scriptedquests.utils.ZoneUtils;
+
+/*
+ * A fragment of a zone; this is used to find zones quickly, but not hold their properties.
+ * Instead, each fragment points to its parent, a zone with properties.
+ * Each zone also keeps track of its fragments.
+ */
+public class ZoneFragment extends BaseZone {
```

Comment:
This is the same comment as in the Zone class. Can you fix one or the other?

Reply:
Done.